exercises/clock/clock_test.go

```diff
@@ -12,15 +12,20 @@ import (
 // (Clock) Add(minutes int) Clock
 //
 // The Add method should also handle subtraction by accepting negative values.
-// To satisfy the readme requirement about clocks being equal, values of
-// your Clock type need to work with the == operator.
+
+// To satisfy the README requirement about clocks being equal, values of
+// your Clock type need to work with the == operator. This means that if your
+// New function returns a pointer rather than a value, your clocks will
+// probably not work with ==.
+//
+// While the time.Time type in the standard library (https://golang.org/pkg/time/#Time)
+// doesn't necessarily need to be used as a basis for your Clock type, it might
+// help to look at how constructors there (Date and Now) return values rather
+// than pointers. Note also how most time.Time methods have value receivers
```

Comment:
wow, I was expecting to ask this to change to "all" instead of "most", but indeed there are some functions with each. I know `go lint` will advise to keep them consistent, but I'm assuming they have a good reason for donig what they've done.

(No change needed for this comment)